source/developer/aarch64.rst

```diff
@@ -0,0 +1,25 @@
+``aarch64`` builds
+==================
+
+.. datechanged:: 2023-11-18
+   Added this section as initial linux-aarch64 builds are starting
+
+While we do not yet have builds for ``osx-arm64`` (see
+:ref:`platform-nomenclature-faq`), we are starting to roll out
+``linux-aarch64`` builds. These run on CircleCI, which offers the
+infrastructure for this (in contrast to GitHub Actions).
+
+This is being initially approached as an opt-in process as we make sure
+all the moving parts are working correctly. A recipe can be flagged for
+building on ``linux-aarch64`` by adding the following to the
```

Comment:
Eventually, yes of course!

Reply:
But not yet? Should we document that? Or should we add an additional key to enable aarch64 containers?
It would be nice to push them if we activate it in the extra section. But I guess that a different discussion.

Reply:
I'm actually unclear on the current status of aarch64 containers. I know there was work on getting base images set up, so it could be that aarch64 containers are already handled. I'm still trying to catch up with the recent work though...I'm sure the answer is in there somewhere and once I figure it out I'll clarify this.

Reply:
OK, looks like there was some discussion on the core chat (Oct 18) about not immediately uploading aarch64 containers. I think this will be a different discussion. I'll add a note here that containers are currently not built and uploaded.

Reply:
Is the core chat public or it is only for the core team ?
I am aware only of https://app.gitter.im/#/room/#bioconda_Lobby:gitter.im but there is no discussion about containers at Oct 18 there.
Thanks!

Reply:
I'd also like to join the discussion about aarch64 support! : )

Reply:
Yeah, there are many discussions in many places; the one I found re: containers was indeed in a separate core team chat. However we should be better about consolidating information and discussion, ideally on a single github issue.

Reply:
@Yikun @martin-g FYI, some recent PRs on pushing aarch64 images to biocontainers: https://github.com/bioconda/bioconda-recipes/pull/43995, https://github.com/bioconda/bioconda-recipes/pull/44457, https://github.com/bioconda/bioconda-utils/pull/941 . . . largely merged into https://github.com/bioconda/bioconda-recipes/pull/40550.